komga-webui/src/components/ThumbnailCard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="col-6 col-lg-3 pa-1">
+    <v-card>
+      <v-img
+        :src="getImage(item)"
+        aspect-ratio="0.7071"
+        contain />
+      <v-card-actions align="center">
+        <v-btn
+          icon
+          disabled>
+          <v-icon>
+            {{ getIcon(item) }}
+          </v-icon>
+        </v-btn>
+        <v-btn
+          icon
+          :color="selected ? 'success' : ''"
+          @click="onClickSelect"
+        >
+          <v-icon>mdi-check</v-icon>
+        </v-btn>
+        <v-btn
+          icon
+          v-if="isDeletable(item)"
+          :color="toBeDeleted ? 'error' : ''"
+          @click="onClickDelete"
+        >
+          <v-icon>mdi-trash-can-outline</v-icon>
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </div>
+
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import {SeriesThumbnailDto} from '@/types/komga-series'
+import {seriesThumbnailUrlByThumbnailId} from '@/functions/urls'
+
+export default Vue.extend({
+  name: 'ThumbnailCard',
+  props: {
+    item: {
+      type:  Object as () => File | SeriesThumbnailDto,
+      required: true,
+    },
+    selected: {
+      type: Boolean,
+      required: true,
+    },
+    toBeDeleted: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  methods: {
```

Comment:
for all the methods you can skip the `function` keyword and use the shorter notation used elsewhere in the project